packages/design-system/src/components/web-components/ds-alert.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './ds-alert';
+
+export default {
+  title: 'Web Components/Alert',
+  argTypes: {
+    children: { control: 'text' },
+    // 'hide-icon': { control: 'boolean' }, uncomment when we support the 'hide-icon' boolean string
+    role: {
+      options: ['alert', 'alertdialog', 'region', 'status'],
+      control: { type: 'radio' },
+    },
+    variation: {
+      options: [undefined, 'success', 'warn', 'error'],
+      control: { type: 'radio' },
+    },
+    weight: {
+      options: [undefined, 'lightweight'],
+      control: { type: 'radio' },
+    },
+  },
+  args: {
+    variation: 'success',
+    heading: 'Yay!',
+  },
+};
+
+const Template = (args) => (
+  <ds-alert {...args} key={JSON.stringify(args)}>
```

Comment:
is the JSON still needed to prevent duplicate children/slots?

Reply:
yeah, without keying on changed props, it won't re-render the whole thing, which means it'll add the "new" children to the previously rendered children. At least that's my theory for what's happening